docs/datetime.md

```diff
@@ -518,7 +547,7 @@ dt2.difference(dt1, { smallestUnit: 'seconds' });
   // => P8456DT12H5M30S
 
 // Months and years can be different lengths
-[jan1, feb1, mar1] = [1, 2, 3].map(month => 
+[jan1, feb1, mar1] = [1, 2, 3].map((month) =>
   Temporal.DateTime.from({ year: 2020, month, day: 1 }));
 feb1.difference(jan1);                                   // => P31D
 feb1.difference(jan1, { largestUnit: 'months' });        // => P1M
```

Comment:
There's a bugfix in #922 on line 527 that should probably be in this PR because it's unrelated to LDT.  Don't remember which commit it's in though.

Reply:
You mean the `may1` -> `mar1` typo?

Reply:
Never mind.  I didn't fix that bug in #922. Instead I opened #921 for it but didn't actually check in a fix for it in datetime.md. Probably makes sense to fix it in this PR though now that we know it's there and we're already editing the DT docs.

Reply:
> You mean the `may1` -> `mar1` typo?

Yep. See comment above that I just posted about this.  Sorry I didn't see your response until I'd already posted. You work fast! ;-)